modules/backlogs/spec/features/work_packages/remaining_time_spec.rb

```diff
@@ -65,10 +65,31 @@
 
     wp_page.update_attributes remainingTime: '125' # rubocop:disable Rails/ActiveRecordAliases
 
-    wp_page.expect_attributes remainingTime: '125'
+    wp_page.expect_attributes remainingTime: '125 h'
 
     work_package.reload
 
     expect(work_package.remaining_hours).to eq 125.0
   end
+
+  it 'displays the remaining time sum properly in hours (Regression #43833)' do
+    work_package
+    wp_table_page = Pages::WorkPackagesTable.new(project)
+
+    query_props = JSON.dump(c: %w(id subject remainingTime),
+                            s: true)
+
+    wp_table_page.visit_with_params("query_props=#{query_props}")
+
+    wp_table_page.expect_work_package_with_attributes work_package, remainingTime: '-'
+
+    wp_table_page.update_work_package_attributes work_package, remainingTime: '125'
+
+    wp_table_page.expect_work_package_with_attributes work_package, remainingTime: '125 h'
+
+    wp_table_page.expect_sums_row_with_attributes remainingTime: '125 h'
```

Comment:
You might want to test this with more than one wp

Reply:
I was only testing the format, not the computation as it is given by the api and I assumed it was already tested.

Digging a little deeper, I found `spec/features/work_packages/index_sums_spec.rb` and maybe I should have added the `remainingTime` check here instead. Do you think it is worth it?